pdns/dnsdistdist/configure.ac

```diff
@@ -8,7 +8,7 @@ AC_CONFIG_HEADERS([config.h])
 AC_PROG_CC
 AC_PROG_CXX
 AC_LANG([C++])
-AC_GNU_SOURCE
```

Comment:
Perhaps we need to add `AC_ENABLE_EXTENSIONS` if we remove this?

Reply:
it is already in the file (a bit higher up)

Reply:
Oops, I meant `AC_USE_SYSTEM_EXTENSIONS`, sorry. I don't see it in dnsdist's configure.ac?

Reply:
It's not there :)